governance/votes/Executive vote - May 30, 2024.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Template - [Executive Vote] SBE Surplus Buffer Upper Limit Update, RWAF and RWA004-A DAO Resolutions, Spark Proxy Spell - May 30, 2024
+summary: Smart Burn Engine Surplus Buffer Upper Limit Update, RWA Foundation (RWAF) and RWA004-A DAO Resolutions, Spark Proxy Spell.
+date: 2024-05-30T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] [Executive Vote] SBE Surplus Buffer Upper Limit Update, RWAF and RWA004-A DAO Resolutions, Spark Proxy Spell - May 30, 2024
+
+The Governance Facilitators and Dewiz have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
```

Comment:
```suggestion
The Governance Facilitators and Dewiz, Sidestream, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
```
@blimpa don't forget the reviewer (not only the crafter) and Phoenix Labs if there is a Spark spell